mail_service/worker_notify_user.py

```diff
@@ -7,24 +7,28 @@
 bs = beanstalkc.Connection(host="172.17.0.1")
 bs.watch("notify_user")
 
+LOGS_DIR = "/srv/pipeline-logs/"
+LOGS_URL_BASE = "https://registry.centos.org/pipeline-logs/"
```

Comment:
+1 to what @mohammedzee1000 said. We already have both these variables defined [here](https://github.com/CentOS/container-pipeline-service/blob/master/beanstalk_worker/constants.py). Please reuse it.